stratisd.conf

```diff
@@ -19,38 +19,6 @@
   <allow send_destination="org.storage.stratis2"
 	 send_interface="org.storage.stratis2.FetchProperties"/>
 
-  <allow send_destination="org.storage.stratis2"
```

Comment:
I think I wasn't clear...please leave the ones for the service in, just remove all the interface-specific ones.

Reply:
Resolved.